convert2rhel/unit_tests/initialize_test.py

```diff
@@ -27,7 +27,8 @@
         (1),
     ),
 )
-def test_run(monkeypatch, exit_code):
+def test_run(monkeypatch, exit_code, tmp_path):
     monkeypatch.setattr(main, "main", value=lambda: exit_code)
+    monkeypatch.setattr(applock, "_DEFAULT_LOCK_DIR", str(tmp_path))
     with pytest.raises(SystemExit):
         initialize.run()
```

Comment:
We probably should have a test for the case where the lock is already held as well since that's the main purpose of this addition.

If you need to examine stderr instead of the logs, here's the pytest documentation for the capsys fixture:
https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function